components/Main/Main.js

```diff
@@ -1,3 +1,9 @@
+import ItemListGeneral from "../ItemListGeneral/ItemListGeneral";
+
 export default function Main() {
-  return <p>Main Part</p>;
+  return (
+    <main>
+      <ItemListGeneral />
+    </main>
```

Comment:
I haven't cloned this repo, so at this point I cannot see where you call the component Main (in the index.js probably). Maybe you need it later for additional purposes but if the only reason for your Main component is to wrap something in main tags, then maybe you can just add those in the index.js? Maybe the coaches have an opinion on this :)

Reply:
correct, the "Main" component is called in pages/index.js.
Hmm..I think both ways work. For me it is more clear as written above. I will ask what's best practice.